### PR TITLE
enable ppc64le builds

### PR DESCRIPTION
**Pull request by jayfurmanek**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Enable builds on ppc64le arch using Travis-ci

### CODE REVIEW

**Comment by jayfurmanek** [comment]
@conda-forge-admin, please rerender

**Comment by jayfurmanek** [comment]
@mcg1969  FYI

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,9 +11,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  skip: true  # [win and py==27]
+  skip: true  # [win and py==27] or [ppc64le and py==27]
```
Comment (dougalsutherland) [inline recipe/meta.yaml]: This isn't really a valid selector ([spec](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=selector#preprocessing-selectors)), and conda-build is actually making a ppc 2.7 build that then doesn't do anything? Pushing a fix.

**Comment by dougalsutherland** [comment]
Will merge when green now; ping me if I forget.

**Comment by jayfurmanek** [comment]
@conda-forge-admin, please rerender

**Comment by jayfurmanek** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (jayfurmanek) [reply recipe/meta.yaml]: yeah - not really sure how to keep smithy from creating the job for 2.7..

Comment (jayfurmanek) [reply recipe/meta.yaml]: re-render fixed it..

**Pull request merged**